RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## v1.7.0
+* Fix misleading dns disabled log
+* Dependency updates
```

Comment:
There are a lot more chnages:
- `--config` is now required when dns is used
- netavark dhcp-proxy correctly renews the lease after dhcp time-out
- bridge: isolate=strict option has been added
- macvlan: bclim option has been added
- no_default_route option has been added
- static routes can now be configured